### ISSUE TITLE
Docker Image is out of date

### ISSUE THREAD

**Issue by DawoudIO** [ChurchCRM/Docker#10]
We are about to release 3.0.0, and it is unclear how to make a new build with this repo. The current build is 2.7.5 which is over a year old.

**Comment by DawoudIO** [comment]
@jaskipper do you recall

**Comment by ajcollett** [comment]
Hey all. I am looking to deploy ChurchCRM for my church, and I am wanting to use docker. I have some fair experience with Docker, so maybe I can help out? Also, I want to point out that ENV variable for passwords is not the best:-P But if you'd like my help, I can make an issue explaining that, and hopefully help fix it.

**Comment by jaskipper** [comment]
Hey @DawoudIO , @ajcollett . I'll take a look at it. Would love any help @ajcollett !

**Comment by ajcollett** [comment]
@jaskipper, I'm going to create a few issues for discussion, then once resolved we can see how I can best contribute.

**Comment by jaskipper** [comment]
@DawoudIO , @ajcollett, @crossan007. I've pushed a new branch (crm-docker-rewrite) with completely updated build files to get Docker working with ChurchCRM 2.10.4. I have split it into 3 different containers: MariaDB, Apache/PHP7 and ChurchCRM. I have also changed over to use Docker Secrets for MYSQL passwords. If you can, test it out and let me know what you think. It's working perfectly on my system. Let me know what changes we could do to enhance it even more. See the Readme for instructions on how to use.

**Comment by crossan007** [comment]
@jaskipper It may be worth documenting where ChurchCRM will keep "uploaded" files such as images.  Do these persist on the docker host, or would images be lost with the docker image?

**Comment by jaskipper** [comment]
@crossan007 Yes. As of now, they are in the containers. We can put them on a separate Docker Volume, which would probably be best. I'll set that up as well.

**Comment by jaskipper** [comment]
I've pushed a new commit (https://github.com/ChurchCRM/Docker/tree/crm-docker-rewrite) that installs SSL by default along with many other changes. See the Readme for installation instructions. I'm getting one error at times in the ChurchCRM interface that says that mod_rewrite has failed, although it seems to be working fine. Any ideas?
